tests/counter/CounterCommands.ts

```diff
@@ -10,105 +9,59 @@ import { CounterDecrementErrorCommand } from "./CounterDecrementErrorCommand.ts"
 import { CounterGetCommand } from "./CounterGetCommand.ts";
 import { CounterIncrementCommand } from "./CounterIncrementCommand.ts";
 
-export function CounterCommands(accounts: Map<string, Account>) {

```

Comment:
I think you could keep the `accounts` as parameter and pass them from the test's entry point (since you do load the accounts anyways in your tests).